Documentation/compilation.rst

```diff
@@ -143,6 +143,10 @@ See section :ref:`Installing ASAGI <installing_ASAGI>`.
 
 .. _compiling-seissol:
 
+Installing easi
+---------------------------
+Follow the `installation instructions <https://easyinit.readthedocs.io/en/latest/index.html>`.
```

Comment:
There is no installation instruction there.
It also says:
>easi is a header-only library written in C++11. Hence, one simply needs (Only unit tests must be compiled with CMake.)
Maybe you want to copy-paste some content from here:
https://gitlab.lrz.de/thomas.ulrich/TuSeisSolScripts/-/blob/master/EasiRelated/README.md

Reply:
Hm, the corresponding PR is already merged but it doesn't seem to be building the documentation.

Reply:
ok, I see.
But don't you think that all users can figure out things like:
```
git clone git@github.com:uphoffc/ImpalaJIT
cd ImpalaJIT 
mkdir build && cd build
CC=mpicc CXX=mpiCC FC=mpif90 cmake ..  -DCMAKE_INSTALL_PREFIX=$HOME
make -j 4
make install
cd ../..
```
(I'm not sure, and detailing may prevent having to give further explanation).

Reply:
It is there, but you need to delete your cache - ctrl+F5 for most browsers.